app/react/Markdown/components/PieChart.js

```diff
@@ -60,17 +60,25 @@ export const PieChartComponent = (props) => {
 
 PieChartComponent.defaultProps = {
   context: 'System',
+  innerRadius: '0',
+  outerRadius: '105',
   classname: '',
   colors: '#ffcc00,#ffd633,#ffe066,#ffeb99,#fff5cc',
   data: null,
+  showLabel: 'false',
```

Comment:
Mmmm... I think I agree with having this default as false, but then that would 'break' the existing Pies.  Any thoughts about that? Would it merit to revert the default?